runner/src/scenario/ceramic/mod.rs

```diff
@@ -67,7 +68,10 @@ impl Credentials {
         let doc: DidDocument = DocumentBuilder::default()
             .id(did)
             .build()
-            .map_err(|e| anyhow::anyhow!("failed to build DID document: {}", e))?;
+            .map_err(|e| anyhow::anyhow!("failed to build DID document: {}", e))
+            .map(|core_doc| {
```

Comment:
why is this needed? shouldn't we already have a `DidDocument` built from ID? Is there a type mismatch or do we not actually need to build it twice (it compiled both ways for me.. so I'm not sure if there's something subtle/annoying going on)? I didn't run a `cargo update`.

Reply:
I was getting conflicting types of DidDocument imports but these somehow resolved itself... can be removed I guess